src/binary/index.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
+const equal = require('./equal');
+const numeric = require('./numeric');
+const add = require('./add');
+const relational = require('./relational');
+
+
+const ASSERTION_TO_EXCLUDE_OPTION_NAME = {
+  [equal.name]: 'equal',
+  [numeric.name]: 'numeric',
+  [add.name]: 'add',
+  [relational.name]: 'relational'
+};
+
+function isExcluded(options, Assertion) {
+  return options.exclude.indexOf(ASSERTION_TO_EXCLUDE_OPTION_NAME[Assertion.name]) >= 0;
+}
+
+
 const assertions = [
-  require('./equal'),
-  require('./numeric'),
-  require('./add'),
-  require('./relational'),
+  equal,
```

Comment:
I suggest to convert `assertions` into object:
```js
const assertions = [
  equal: require('./equal'),
  numeric : require('./numeric'),
  add: require('./add'),
  relational: require('./relational'),
];
```
This avoids adding constant `ASSERTION_TO_EXCLUDE_OPTION_NAME`.